src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -185,11 +185,20 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
 
         protected override async Task<bool> HandleUnauthorizedAsync(ChallengeContext context)
         {
-            var authResult = await HandleAuthenticateOnceAsync();
+            Exception failure = null;
```

Comment:
In Katana we had global try/catch blocks and it made the components impossible for users to debug. We explicitly removed them in K and tried to restrict our exception handling to expected exceptions (e.g. token validation).